icc-api/api/XHR.ts

```diff
@@ -114,7 +114,7 @@ export namespace XHR {
         ? response.json()
         : ct.startsWith("application/xml") || ct.startsWith("text/")
         ? response.text()
-        : response.arrayBuffer()
+        : response.blob().then(blob => new Response(blob).arrayBuffer())
```

Comment:
Please verify that the function `arrayBuffer` does not exists and only convert to blob only if needed.